spec/system/panels/submissions/reviews_spec.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+include UsersHelper
+
+RSpec.describe 'Panel Submission Reviews', type: :system, js: true do
+  let!(:grant)      { create(:grant_with_users) }
```

Comment:
suggest:
`  let!(:grant)      { create(:published_grant, :with_users_and_submission_form, :with_panel) }`
see https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md#using-traits for using traits while constructing instances